main.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // Version holds the current Gitea version
-const Version = "0.9.99.0915"
+var Version = "v0.0.0"
```

Comment:
But than go get doesn't provide any version. And just because git spits that out is not a reason to blindly take it.